jsapp/js/components/modalForms/bulkEditSubmissionsForm.es6

```diff
@@ -173,10 +176,23 @@ class BulkEditSubmissionsForm extends React.Component {
    * @returns {object[]} a list of question objects with all responses included
    */
   getDisplayData() {
-    const questions = getFlatQuestionsList(this.props.asset.content.survey);
+    let questions = getFlatQuestionsList(this.props.asset.content.survey);
     const flatPaths = getSurveyFlatPaths(this.props.asset.content.survey);
 
-    questions.forEach((question) => {
+    questions = questions.filter((question) => {
+      // we don't want to include special calculate row used to store form version
+      if (question.type === QUESTION_TYPES.calculate.id && question.name === FORM_VERSION_NAME) {
```

Comment:
There is a small edge case that bypasses this check: If a form has multiple versions like [this one](https://github.com/kobotoolbox/kpi/files/6131190/aEYQ8CkLw3QKmynaXUAw8U.xlsx) for example

Reply:
This is a great comment. It actually makes more sense to just hide all `calculate` rows here (and in the single submission modal)!